util/cleanup.h

```diff
@@ -25,6 +27,18 @@ static inline void freep(void *p)
 }
 #define _cleanup_free_ __cleanup__(freep)
 
+static inline void free_hugep(struct nvme_mem_huge **mh)
+{
+	nvme_free_huge(*mh);
+}
+#define _cleanup_free_huge_ __cleanup__(free_hugep)
```

Comment:
I don't see any users of this. Are there plans to add some in the future?

Reply:
I was not sure if I should leave it around even there is no user right now. It's fairly simple to add if needed anyway.